decidim-core/app/controllers/decidim/open_data_controller.rb

```diff
@@ -21,8 +21,8 @@ def download
 
     private
 
-    def open_data_core
-      [:users, :user_groups]
+    def open_data_core_manifests
+      @open_data_core ||= Decidim.open_data_manifests.select(&:include_in_open_data)
```

Comment:
I think the convention for memoizations is that the variable should use the same name as the method
```suggestion
      @open_data_core_manifests ||= Decidim.open_data_manifests.select(&:include_in_open_data)
```